### ISSUE TITLE
Major changes 2023-06-10 Release candidate

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-releases#36]
Rat + 18456 IEAs rat 
C. elegans + 2168 IEAs 

Ensembl -54690 / 272420 | -20.08% (lost 95k dog annotations) 
InterPro | -15000 / 223274 | -6.72%  (corresponds to the number of Dig InterPro lost
GOC | -4360 / 31551 | -13.82%

NCBITaxon:9615\|Canis lupus familiaris | -135303 / 99056 | -136.59%

**Comment by pgaudet** [comment]
Emailed Alex about dog data: 

Hi Alex, 

How are you ? I am doing QC on the GO release, and it seems we have lost most dog IEA annotations (80%): we used to have 166k, now we only have 31610. Is this expected? 

I am not sure I would block the release for this (unless you tell me otherwise) – it seems this genome has been really unstable. 

Thanks, Pascale


____

Alex's reply: 

Unfortunately, yes. The genome, which UniProt getting from the Ensembl, was changed again, and I expect the annotation to recover later this year.
